config/routes.rb

```diff
@@ -3,12 +3,13 @@
     member do
       delete :purge_content
     end
+    resources :chargings, :only => [:create, :destroy]
   end
+  patch '/cars/:car_id/chargings' => "chargings#update"
```

Comment:
Maybe, it would make more sense to nest chargings and repairs inside cars like we will do with taxes

Reply:
I had to override the update path, because the form with method: :create leads to '/cars/:car_id/chargings', but on update the :create method gets overridden by the Update button with formmethod="patch" which leads to the same url '/cars/:car_id/chargings'
The update default url is '/cars/:car_id/chargings/:id'.

If the hidden field is a security issue then the solution I can think of is to make an AJAX request to Update with its default url '/cars/:car_id/chargings/:id'

Reply:
maybe that's because you missed `:update` property in line 6?
try like this: `resources :chargings, :only => [:create, :update, :destroy]`